db/controllers/comments.controller.js

```diff
@@ -15,9 +19,20 @@ exports.getComments = (req, res, next) => {
 exports.postComment = (req, res, next) => {
   const { username, body } = req.body;
   const { article_id } = req.params;
+
   insertCommment(username, body, article_id)
     .then((postedComment) => {
       res.status(201).send({ postedComment });
     })
     .catch(next);
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  Promise.all([destroyComment(comment_id), commentChecker(comment_id)])
+    .then(() => {
+      res.status(204).send();
+    })
```

Comment:
Good job with this! You have implemented the functions correctly and handled them with a Promise.all 👍